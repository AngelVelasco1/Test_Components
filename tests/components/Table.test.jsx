import { render } from "@testing-library/react";
import Table from "../../src/components/Table";

describe("table component", () => {
    test("Snapshoot of component", () => {
        let {container} = render(<Table/>);
        expect(container).toMatchSnapshot();
    });
})