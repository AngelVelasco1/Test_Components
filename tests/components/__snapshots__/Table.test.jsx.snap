// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`table component Snapshoot of component 1`] = `
<div>
  <table>
    <ul
      class="nav nav-tabs  "
    >
      <li
        class="nav-item"
      >
        <a
          aria-current="page"
          class="nav-link active"
          href="#"
        >
          Active
        </a>
      </li>
      <li
        class="nav-item"
      >
        <a
          class="nav-link"
          href="#"
        >
          Link
        </a>
      </li>
      <li
        class="nav-item disabled"
      >
        <a
          class="nav-link"
          href="#"
        >
          Disabled
        </a>
      </li>
    </ul>
  </table>
</div>
`;
